src/main/java/Cheryl.java

```diff
@@ -0,0 +1,50 @@
+import cheryl.util.FileSystem;
+import cheryl.ui.Manager;
+import cheryl.ui.UI;
+
+/**
+ * Represents the overall chatbot
+ * Provides methods to begin running the chatbot.
+ *
+ * @author Nithvin Leelakrishnan
+ * @version 1.0
+ */
+public class Cheryl {
+    private final Manager manager;
+
+    /**
+     * Constructs a new Cheryl instance.
+     * Initializes the manager with tasks loaded from the file system.
+     */
+    Cheryl() {
+        this.manager = new Manager(new FileSystem().getTasks());
+    }
+
+    /**
+     * Runs the main loop until user gives exit command.
+     */
+    public void run() {
+        UI.printIntro();
+        Boolean runMainLoop = true;
```

Comment:
Hello! A small nitpick here is that the coding standard recommends that Boolean variables should be named to sound like booleans: sound like a question.
```suggestion
        Boolean shouldRunMainLoop = true;
```